meta-ostro/classes/ostro-image.bbclass

```diff
@@ -230,15 +230,15 @@ ostro_root_authorized_keys () {
 NOISO = "1"
 
 # Add support for compressing images with zip. Works for arbitrary image
-# types. Example: OSTRO_VM_IMAGE_TYPES = "dsk.zip dsk.vdi.zip"
```

Comment:
we probably don't want to. if there is ova, there is no need on vdi.
Or, if somebody don't want to use ova but still use VBox, he should have enough knowledge to how to convert between raw images to something else.